src/affinder/_tests/test_affinder.py

```diff
@@ -137,3 +137,19 @@ def test_apply_affine_nonimage():
     widget = apply_affine()
     with pytest.raises(NotImplementedError):
         widget(ref_layer, mov_layer)
+
+
+def test_load_affine():
+    existing_affine = np.array([[0.9, 0.1, 5], [0.4, 0.2, 9], [0, 0, 1]])
+
+    layer = napari.layers.Image(np.random.random((5, 5)))
+    layer.affine = existing_affine
+
+    widget = load_affine(,
```

Comment:
I think I found the problem in your tests @andreasmarnold 😂 

btw, rather than making a small test file, you might as well make it within the test using a temporary file. You can use a pytest fixture called `tmp_path` and this will give you a pathlib.Path pointing to a temporary directory that exists for the duration of the test. So you would do something like this:

```python
def test_load_affine(tmp_path):
    affile = tmp_path / 'affine.txt'
    affine = np.array([[2, 0, 50], [0, 2, 10], [0, 0, 1]], dtype=float)
    np.savetxt(affile, affine)

    widget = load_affine()  # ;-)
    ...
```

make sense?

Reply:
Well ... _mea culpa_ 😂